app/helpers/list_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ListHelper
+end
```

Comment:
Did you use a generator George? Seems like this helper, `list.js` and `list.scss` are unnecessary.